lib/matplotlib/font_manager.py

```diff
@@ -1278,12 +1278,15 @@ def _rebuild():
 try:
     fontManager = json_load(_fmcache)
 except Exception:
+    print("Rebuilding font cache.")
```

Comment:
This code is executed at import time. I don't think we should pollute stdout with such messages. Maybe `_log.info()` or `_log.debug()`?

Reply:
I'm just trying to debug the failure in Travis (this is a separate commit, in fact) as I can't repro the failure locally.